packages/scipy/patches/fix-implicit-cast-args-from-newer-lapack.patch

```diff
@@ -0,0 +1,23 @@
+We are hijacking dfft.f to inject missing functions from future versions of LAPACK.
+
+This patch is applied to the generated dfft.c after f2c is applied to dfft.f. 
+This must be done in between when f2c is applied to dfft.f and when dfft.c is compiled.
+This is arranged in _f2c_fixes.py.
```

Comment:
Sigh, I'm eagerly waiting for lfortran to be able to compile scipy for wasm.

Reply:
I think this is also caused by bad merge. This patch isn't used.